pkg/controller/certificates/sync.go

```diff
@@ -353,7 +353,7 @@ func (c *Controller) issue(ctx context.Context, issuer issuer.Interface, crt *v1
 		return err
 	}
 
-	if resp.PrivateKey == nil {
+	if resp == nil {
```

Comment:
This change will technically allow issuers to return *just* a certificate without a corresponding private key, which is probably not desired.

None of our issuers do this at the moment, but may be something we want to gate against in future.